opennms-webapp/src/main/webapp/js/angular-onms-elementList-minion.js

```diff
@@ -64,6 +66,29 @@
 				function(value, headers) {
 					$scope.$parent.items = value;
 
+					if (value && value.length > 0) {
+						var query = '(' + value.map(function(minion) {
```

Comment:
Should we also be matching on the `foreignSource` name here? Since it is technically possible for a different node in a another requisition to have the same id.

Reply:
Yeah, I guess just assume the 'Minions' foreign source hardcoded?